scripts/sdk/cosmwasm/multiswap.js

```diff
@@ -34,23 +34,6 @@ class MultiswapContract {
     return owner;
   }
 
-  async getFee(token) {
-    let wallet = await DirectSecp256k1HdWallet.fromMnemonic(this.mnemonic, {
-      prefix: "cudos",
-    });
-    let client = await SigningCosmWasmClient.connectWithSigner(
-      this.rpcEndpoint,
-      wallet
-    );
-    const fee = await client.queryContractSmart(this.contract, {
-      fee: {
-        token,
-      },
-    });
-    console.log("fee", fee);
-    return fee;
-  }
```

Comment:
This getFee function should be kept - since it's a required function for fee management.